bsp4s/src/main/scala/ch/epfl/scala/bsp/endpoints/Endpoints.scala

```diff
@@ -45,6 +45,8 @@ trait BuildTarget {
   object scalaMainClasses
       extends Endpoint[ScalaMainClassesParams, ScalaMainClassesResult](
         "buildTarget/scalaMainClasses")
+  object scalaClasspath extends Endpoint[ExecutionEnvironmentParams, ExecutionEnvironmentResult]("buildTarget/executionEnvironment")
```

Comment:
```suggestion
  object executionEnvironment extends Endpoint[ExecutionEnvironmentParams, ExecutionEnvironmentResult]("buildTarget/executionEnvironment")
```
Or `javaExecutionEnvironment`

Reply:
Maybe `jvmExecutionEnvironment` would be more appropriate? The execution environment for JVM languages should be the same.

Reply:
Changed it to jvmTestEnvironment, according to the previous comment